conda-recipes/tensor2tensor-feedstock/LICENSE

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
```

Comment:
This should be the Apache License from github. The Licensing terms of Apache are preferred by IBM over BSD-3.

Reply:
You mean Apache License from tensor2tensor's github?